source/ast/expressions/AssertionExpr.cpp

```diff
@@ -1125,6 +1464,28 @@ void ConditionalAssertionExpr::serializeTo(ASTSerializer& serializer) const {
         serializer.write("else", *elseExpr);
 }
 
+/// Returns maximum possible sequence length beetween all case labels sequence expressions
+/// include `default` label
+std::optional<SequenceRange> CaseAssertionExpr::computeSequenceLengthImpl() const {
+    std::set<SequenceRange> seqRangeItems;
```

Comment:
You're just using the set for sorting here right? It's generally a good idea to avoid allocating containers unless strictly necessary, and in this case I think you can just store the current maximum and update it on each iteration if the new value is larger? You don't actually need to store all the items.

Reply:
fixed